build.gradle

```diff
@@ -1,8 +1,270 @@
+version = '1.13.1.0'
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+bundlePlugin {
+    archiveName("opendistro-security-${version}.zip")
+
+    from("securityconfig") {
+        into "securityconfig"
+    }
+    from("tools") {
+        into "tools"
+    }
+    from("plugin-security.policy") {
+        into "."
+    }
+}
+
+group = 'com.amazon.opendistroforelasticsearch'
+version = '1.13.1.0'
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+apply plugin: 'distribution'
+distributions {
+    custom {
+        distributionBaseName = "opendistro-security-${version}-securityadmin-standalone"
+        contents {
+            into('./tools/') {
+                from 'tools'
+            }
+            into('./deps/securityconfig') {
+                from 'securityconfig'
+            }
+        }
+    }
+    customDistZip.setArchiveName("opendistro-security-${version}-securityadmin-standalone.zip")
+    customDistTar.setArchiveName("opendistro-security-${version}-securityadmin-standalone.tar.gz")
+}
+
+customDistTar{
+    compression = Compression.GZIP
+    archiveExtension = "tar.gz"
+}
+
+ext {
+    projectSubstitutions = [:]
+    licenseFile = rootProject.file('LICENSE.txt')
+    noticeFile = rootProject.file('NOTICE.txt')
+}
+
+configurations.all {
+    if (it.state != Configuration.State.UNRESOLVED) return
+    resolutionStrategy {
+        force 'commons-codec:commons-codec:1.14'
+        force 'org.slf4j:slf4j-api:1.7.26'
+        force 'org.apache.httpcomponents:httpclient:4.5.10'
+        force 'com.google.code.findbugs:jsr305:3.0.2'
+        force 'com.fasterxml.jackson.core:jackson-databind:2.11.2'
+        force 'com.fasterxml.jackson.core:jackson-core:2.11.2'
+        force 'commons-cli:commons-cli:1.3.1'
+        force 'commons-logging:commons-logging:1.2'
+        force 'com.google.guava:guava:25.1-jre'
+        force 'org.bouncycastle:bcprov-jdk15on:1.67'
+        force 'org.apache.commons:commons-lang3:3.4'
+        force 'org.apache.santuario:xmlsec:2.2.0'
+        force 'com.fasterxml.woodstox:woodstox-core:6.2.1'
+        force 'io.netty:netty-buffer:4.1.49.Final'
+        force 'io.netty:netty-common:4.1.49.Final'
+        force 'io.netty:netty-handler:4.1.49.Final'
+        force 'io.netty:netty-transport:4.1.49.Final'
+        force 'com.fasterxml.jackson.core:jackson-annotations:2.11.2'
+        force 'net.sf.jopt-simple:jopt-simple:5.0.4'
+        force 'org.scala-lang:scala-library:2.12.10'
+        force 'org.scala-lang:scala-reflect:2.12.10'
+        force 'org.cryptacular:cryptacular:1.1.4'
+        force 'org.apache.httpcomponents:httpcore:4.4.12'
+        exclude group: "org.elasticsearch", module: "securemock"
+    }
+}
+
+configurations.testCompile {
+    if (it.state != Configuration.State.UNRESOLVED) return
+    resolutionStrategy {
+        force 'org.apache.kafka:kafka-clients:2.0.1:test'
+    }
+}
+
+dependencies {
+    compile('org.elasticsearch.plugin:transport-netty4-client:7.10.2') {
```

Comment:
`compile` is deprecated in favor of `implementation`. Can we use that instead?

Reply:
Replacing all `compile` with `implementation` fails `compileJava` task. Some of them indeed can be replaced. I didn't change them for consistency.